packages/design-system/src/styles/settings/variables/_core-defaultTheme.scss

```diff
@@ -0,0 +1,145 @@
+$animation-ease-in-out-expo: cubic-bezier(1, 0, 0, 1) !default;
```

Comment:
This one as well can be removed.

Reply:
Nevermind I see the other theme files, I'll remove these :+1: